fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -1125,7 +1125,7 @@ public static void loadJournal(Env env, Long logId, JournalEntity journal) {
                     break;
                 }
                 case OperationType.OP_DELETE_TABLE_STATS: {
-                    env.getAnalysisManager().replayTableStatsDeletion((TableStatsDeletionLog) journal.getData());
+                    LOG.info("replay delete table stat: {}", ((TableStatsDeletionLog) journal.getData()).id);
```

Comment:
Without replay table stats deletion, how to clear a follower's table stats?

Reply:
> Without replay table stats deletion, how to clear a follower's table stats?

Thanks for you comment, i hava updated the code.